lte/gateway/c/core/oai/test/spgw_task/mock_spgw_op.h

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <vector>
+
+#include "spgw_state_converter.h"
+
+namespace magma {
+namespace lte {
+
+#define DEFAULT_SPGW_CONTEXT_DATA_PATH                                         \
+  "/home/vagrant/magma/lte/gateway/c/core/oai/test/spgw_task/data/"
```

Comment:
I would suggest getting the user's home directory instead and prepend it instead of harcoding `/home/vagrant/`
```
    homedir = getpwuid(getuid())->pw_dir;
```

(or better yet, can we get the relative path??)